cowait/tasks/graph/graph_task.py

```diff
@@ -0,0 +1,57 @@
+import asyncio
+from cowait.tasks import Task
+from .graph import Graph
+
+
+class GraphTask(Task):
+    async def define(self, graph, **inputs):
+        # this is where you would define your graph nodes
+        # to create a dag, override this function in a subclass
+        pass
+
+    async def run(self, **inputs):
+        g = Graph()
+        await self.define(g, **inputs)
+
```

Comment:
Before we run this we should make sure it has no loops

Reply:
a validation step sounds like a pretty good idea 😬